start_integration.py

```diff
@@ -22,8 +27,9 @@
     ihub_data = json.loads(PFile.read().decode('utf-8'))
 
 process_id = ihub_data['processId']
+log_level = ihub_data['logLevel']
 
 notification_service = SmsNotifService(service_id, process_id, ov_url, ov_login, ov_pwd, phone_number_field,
-                                       awsAccessKeyId, awsSecretAccessKey, awsRegion, max_attempts,
+                                       awsAccessKeyId, awsSecretAccessKey, awsRegion, log_level, max_attempts,
```

Comment:
I propose to move the `process_id` and `log_level` parameters to the end, and follow this convetion in other integrations.

Reply:
Done